libs/version/fixed_makefile_part

```diff
@@ -0,0 +1,42 @@
+# Returns the number corresponding to the specified blocksds version.
```

Comment:
This file should have the extension `.make` to help editors highlight the code.

Reply:
Did not know that. Nice!